src/App.js

```diff
@@ -1,14 +1,32 @@
 /* eslint class-methods-use-this: ["error", { "exceptMethods": ["render"] }] */
 /* eslint-env es6 */
```

Comment:
- [ ] @SergSkol great work on your project however it's not good practice to disable linter checks in your file, it reverses the importance of linter checks in helping us write quality code. Kindly try and solve the linter errors instead of disabling them.

Reply:
Thank you. I've deleted this eslint-ignore